tools/bug-report/pkg/kubectlcmd/kubectlcmd.go

```diff
@@ -0,0 +1,130 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubectlcmd
```

Comment:
This whole package is re-implementing pkg/kube

cc @nmittler who spent a lot of time cleaning up other packages reinventing pkg/kube

Reply:
it's re-implementing just the needed part which is only a couple of functions. i really want to avoid taking on that whole framework which afaics is only used for integration tests and has way more than the simple functions needed here.
i had a closer look at the framework and there is one really nice thing about - it does use go client under the hood. i'm not sure if we'll see much performance improvement in practice but it seems reasonable to use it because of that.

Reply:
I am not sure its only a couple of functions. You have fetching proxy logs, arbitrary execs, fetching proxy information, fetching istiod information, etc.

This is not just used for testing, its already used for every single other istioctl command, and even Istiod